bin/compile

```diff
@@ -1,31 +1,17 @@
-#!/usr/bin/env ruby
+#!/bin/bash
 
-# sync output
-$stdout.sync = true
+BIN_DIR=$(cd $(dirname $0); pwd)
+ROOT_DIR=$(dirname $BIN_DIR)
 
-$:.unshift File.expand_path("../../lib", __FILE__)
-require "language_pack"
-require "language_pack/shell_helpers"
+ruby_dir="$ROOT_DIR/vendor/ruby/$STACK"
```

Comment:
Pragmatically i've never actually seen a ticket where this has bit us in the butt so i'm fine moving forward with this. Ideologically I'm not in love with it. At the end of the day i'm fine with this location.